luigi/tools/range.py

```diff
@@ -101,6 +101,9 @@ class RangeBase(luigi.WrapperTask):
     now = luigi.IntParameter(
         default=None,
         description="set to override current time. In seconds since epoch")
+    param_name = luigi.Parameter(
+        default=None,
```

Comment:
Maybe add `positional=False`. It does not need it more than the other parameters, but one must start somewhere.